airbyte-integrations/connectors/source-s3/unit_tests/v4/test_legacy_config_transformer.py

```diff
@@ -209,6 +218,64 @@ def test_convert_legacy_config(legacy_config, expected_config):
             ValueError,
             id="test_malformed_advanced_options",
         ),
+        pytest.param(
+            "csv",
+            {
+                "advanced_options": '{"column_names": ""}',
+            },
+            None,
+            ValueError,
+            id="test_unsupported_advanced_options",
+        ),
+        pytest.param(
+            "csv",
+            {
+                "advanced_options": '{"check_utf8": false}',
+            },
+            {
+                "filetype": "csv",
+                "delimiter": ",",
+                "quote_char": '"',
+                "encoding": "utf8",
+                "double_quote": True,
+                "null_values": ["", "null", "NULL", "N/A", "NA", "NaN", "None"],
```

Comment:
No action item yet but I did change [those three default values](https://github.com/airbytehq/airbyte/pull/29541/files#diff-1eb39ab500396d5774c982a7f8367724c40d42458456bc1f224f70b04a449354R83-R85) in the rollout branch so this will have to be updated once we merge this into the rollout branch

Reply:
I was going to merge it into my other PR and then into master, instead of into the rollout branch, to isolate the rollout branch to the switchover instead of including bug fixes. WDYT?